<launch> 

    <!-- Launch State Publisher -->
    <node pkg="rover" type="state_publisher.py" name="state_publisher_node" output="screen"/>

    <!-- Launch GUI -->
    <!-- <node pkg="rover" type="rover_gui.py" name="GUI" output="screen"/> -->

    <!-- Launch Main State Machine Node -->
    <node pkg="rover" type="state_machine.py" name="state_machine_node" output="screen">
        <param name="num_gps_goals" value="7"/>
    </node>
    <!-- Launch Auxillary Nodes -->
    <node pkg="rover" type="gps_goal_checker.py" name="gps_checker_node" output="screen"/>
    <node pkg="rover" type="nuc_comms_check.py" name="comms_check_node" output="screen"/>

</launch>
